liquidhaskell-boot/src/Language/Haskell/Liquid/GHC/Misc.hs

```diff
@@ -931,7 +931,7 @@ withWiredIn m = discardConstraints $ do
   --   ) wiredIns
 
   sigs wiredIns = concatMap (\w ->
-      let inf = maybeToList $ (\(fPrec, fDir) -> Ghc.L locSpanAnn $ Ghc.FixSig Ghc.noAnn $ Ghc.FixitySig Ghc.NoNamespaceSpecifier [Ghc.L locSpanAnn (tcWiredInName w)] $ Ghc.Fixity Ghc.NoSourceText fPrec fDir) <$> tcWiredInFixity w in
+      let inf = maybeToList $ (\(fPrec, fDir) -> Ghc.L locSpanAnn $ Ghc.FixSig Ghc.noAnn $ Ghc.FixitySig noExtField [Ghc.L locSpanAnn (tcWiredInName w)] $ Ghc.Fixity Ghc.NoSourceText fPrec fDir) <$> tcWiredInFixity w in
```

Comment:
This difference would not be necessary if Compat offers a function to build FixSig.

Reply:
Yes, of course. The reason I didn't want to go through all these is because it's very ad-hoc, and on `develop` you wouldn't know when to add a new one anyway, so wouldn't this kind of stuff be better captured in the individual merge commits from `develop` into `ghc-9.8-branch`, for the merges that introduce new such uses of the GHC API?

Reply:
I'm proposing the change only to reduce conflicts when rebasing or cherry picking commits. If anyone were to modify some other part of this line, it would cause a conflict between the branches.

> on develop you wouldn't know when to add a new one anyway

It is possible that when upgrading compilers and there is a definition change like this, the author could put it in Compat instead of doing the modification in place.

But I think it's fine to defer on these if the value is not evident yet.